Documentation/applications/interpreters/wamr/index.rst

```diff
@@ -1,3 +1,53 @@
 ==================================
 ``wamr`` Webassembly Micro Runtime
 ==================================
+
+This guides explains how to try WAMR on NuttX.
+
+Install WASI SDK
+================
+
+We need be able to build WASM programs. this can be done by installing the ``wasi-sdk`` as shown in the `WAMR guide <https://wamr.gitbook.io/document/basics/getting-started/host_prerequsites>`_.
```

Comment:
```suggestion
We need to be able to build WASM programs. This can be done by installing the ``wasi-sdk`` as shown in the `WAMR guide <https://wamr.gitbook.io/document/basics/getting-started/host_prerequsites>`_.
```